core/include/mutex.h

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
- *               2013, 2014 Freie Universität Berlin
+ *               2013-2017 Freie Universität Berlin
```

Comment:
Nitpick, but shouldn't it be `2013, 2014, 2017`?

Reply:
fixed.